lightning/cookie.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import annotations
+
+from datetime import datetime
+from lightning_adapter.models import Cookie as RawCookie
+
+
+class Cookie(dict):
+
+    def __init__(self, *, name: str, value: str, path: str = None,
+                 domain: str = None, secure_only: bool = False,
+                 http_only: bool = False, expires: datetime.timestamp = None,
+                 same_site_policy: str = None):
+
+        self._raw = RawCookie(name=name, value=value, path=path,
+                              domain=domain, secure=secure_only, http_only=http_only,
+                              expiry=expires, same_site=same_site_policy
+                              )
+
+    def raw(self):
```

Comment:
Add type hints to every method below.

Reply:
fixed in f6c8cafd410d4a3b618d6c8574c6e07b1b8028c0